embedded-openshift/README.md

```diff
@@ -0,0 +1,70 @@
+Example of using Embedded Datagrid on OpenShift
+==================================================================
+Author: Katia Aresti
+Level: Intermediate
+Technologies: Infinispan, OpenShift
+Summary: The `embedded-openshift` quickstart demonstrates how to deploy an application that uses datagrid in
+embedded mode
+Target Product: JDG
+Product Versions: JDG 7.3
+
+What is it?
+-----------
+
+The `embedded-openshift` quickstart demonstrates how to use and deploy applications using datagrid in embedded mode.
+
+Prerequisites
+-------------
+
+In order to run this project one needs to have an access to local OpenShift installation.
+
+You can use local OpenShift all-in-one cluster. To prepare you local OpenShift cluster follow [the documentation](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md).
+
+Maven and [Docker](https://www.docker.com/) daemon running in the background.
+
+System requirements
+-------------------
+
+All you need to build this project is Java 8.0 (Java SDK 1.8) or better, Maven 3.0 or better and OpenShift 3 client.
+
+Configure Maven
+---------------
+
+If you have not yet done so, you must [Configure Maven](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/CONFIGURE_MAVEN.md#configure-maven-to-build-and-deploy-the-quickstarts) before testing the quickstarts.
+
+Configuring Service Account for KUBE_PING
+-----------------------------------------
+Since https://github.com/jgroups-extras/jgroups-kubernetes[KUBE_PING] Discovery Protocol requires viewing all Pods in the OpenShift Project, you need to add additional privileges to the OpenShift:
+
+
+      oc cluster up
+      oc login -u system:admin
+      oc policy add-role-to-user view system:serviceaccount:myproject:default -n myproject
```

Comment:
Correct me if I'm wrong but I believe you can add the view role to the default service account as the `developer` user because it applies to the current project namespace. Also consider this alternative:
`$ oc policy add-role-to-user view system:serviceaccount:$(oc project -q):default -n $(oc project -q)`

Reply:
Ok ! I'm going to test it as you tell me. I took this from kubernetes/openshift example we have in upstream. We might need to update the docs there too.
https://github.com/infinispan/infinispan-simple-tutorials/blob/master/kubernetes/README.adoc